controllers/apply-next/views/startpage.njk

```diff
@@ -12,10 +13,35 @@
                 suffix = copy.description | safe
             ) }}
 
-            <h2>Introduction</h2>
-            <p>Form intro goes here</p>
+            {% if applications %}
```

Comment:
We'll probably need to distinguish between the "dashboard" page, where your applications live, and the "startpage"  which summaries the form and includes basic guidance. This is fine for now though.

Reply:
Yeah – I figured at this point we only want to show you applications for *this* form, but yeah, we'll tweak this later.